modin/pandas/window.py

```diff
@@ -272,3 +272,128 @@ def quantile(self, quantile, interpolation="linear", **kwargs):
+    def __init__(self, dataframe, min_periods=1, center=None, axis=0, method="single"):
+        self._dataframe = dataframe
+        self._query_compiler = dataframe._query_compiler
+        self.expanding_args = [
+            min_periods,
+            center,
+            axis,
+            method,
+        ]
+        self.axis = axis
+
+    def aggregate(self, func, *args, **kwargs):
+        from .dataframe import DataFrame
+
+        dataframe = DataFrame(
+            query_compiler=self._query_compiler.expanding_aggregate(
+                self.axis, self.expanding_args, func, *args, **kwargs
+            )
+        )
+        if isinstance(self._dataframe, DataFrame):
+            return dataframe
+        elif is_list_like(func):
+            dataframe.columns = dataframe.columns.droplevel()
+            return dataframe
+        else:
+            return dataframe.squeeze()
+
+    def sum(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_sum(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def min(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_min(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def max(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_max(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def mean(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_mean(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def var(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_var(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def std(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_std(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def count(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_count(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def sem(self, *args, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_sem(
+                self.axis, self.expanding_args, *args, **kwargs
+            )
+        )
+
+    def skew(self, **kwargs):
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.expanding_skew(
```

Comment:
@noloerino, I do not see that the QC has `expanding_skew`, `expanding_kurt` interfaces (and probably some others). Why is that?

Reply:
Looks like @noloerino forgot to test those, and we didn't add test_expanding to CI, so we never caught the error. I'll fix these in #6156.